### ISSUE TITLE
Blocked scanlation groups still show as next chapter when reading manga

### ISSUE THREAD

**Issue by kyjibo** [CarlosEsco/Neko#1169]
### Steps to reproduce

Pick a manga with chapters from multiple scanlators.
Block one of the groups
Starting from a chapter by an unblocked group right before a chapter that is now blocked, scroll to the end of the chapter.

### Expected behavior

Next: shows the first following unblocked chapter

### Actual behavior

Next: shows the following blocked chapter

### Neko version

2.10.13

### Android version

Android 12

### Device

Samsung Galaxy S10e

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
2.10.14
